app/models/study_file.rb

```diff
@@ -83,6 +83,8 @@ class StudyFile
   field :z_axis_min, type: Integer
   field :z_axis_max, type: Integer
   field :is_spatial, type: Boolean, default: false
+  # for spatial files, the ids of cluster files that correspond to this file for default display
+  field :spatial_cluster_associations, type: Array, default: []
```

Comment:
I feel like there's a more robust way of managing this association, but given since they're both `StudyFile` instances of type `Cluster`, nothing is coming to mind.  But we should ruminate on this in the hopes that something may shake out.